app/src/main/java/com/nextcloud/client/network/ClientFactoryImpl.java

```diff
@@ -37,20 +37,20 @@
 
 import java.io.IOException;
 
-class ClientFactoryImpl implements ClientFactory {
+public class ClientFactoryImpl implements ClientFactory {
```

Comment:
```suggestion
class ClientFactoryImpl implements ClientFactory {
```
Not needed anymore since injection works on the WidgetProvider. Same for the constructor, but GH won't let me add a suggestion there.

Reply:
This is needed by BitmapIT.

Reply:
Then add @VisibleForTesting